content/docs/manuals/app-catalog/nginx-server.md

```diff
@@ -0,0 +1,199 @@
+#### SLA Template
+
+For this example, we create a service named `curlv4` using a `curlimages/curl:7.82.0` docker image. This service performs a curl request to an Okestra semantic IP address of our choice (`10.30.55.55`), then fails. After failure, Oakestra will re-deploy the instance indefinitely.
+#### SLA Template
+#### SLA Template
+
+For this example, we create a service named `curlv4` using a `curlimages/curl:7.82.0` docker image. This service performs a curl request to an Okestra semantic IP address of our choice (`10.30.55.55`), then fails. After failure, Oakestra will re-deploy the instance indefinitely.
+
+{{< callout context="tip" title="Oakestra Dashboard" icon="outline/rocket">}}
+  You can choose the IPv4 addresses for each service arbitrarily. In your deployment descriptor, you can type an IP address in the form `10.30.X.Y` next to the associated balancing policy. In this example,` "rr_ip": "10.30.55.55"`  binds `10.30.55.55` to the round-robin policy for that service. Every time we perform a request towards that address, the packets are balanced across its instances using round-robin balancing. More details are available in the Networking section of the manuals. 
+{{< /callout >}}
+
+For this example, we create a service named `curlv4` using a `curlimages/curl:7.82.0` docker image. This service performs a curl request to an Okestra semantic IP address of our choice (`10.30.55.55`), then fails. After failure, Oakestra will re-deploy the instance indefinitely.
+
+{{< callout context="tip" title="Oakestra Dashboard" icon="outline/rocket">}}
+  You can choose the IPv4 addresses for each service arbitrarily. In your deployment descriptor, you can type an IP address in the form `10.30.X.Y` next to the associated balancing policy. In this example,` "rr_ip": "10.30.55.55"`  binds `10.30.55.55` to the round-robin policy for that service. Every time we perform a request towards that address, the packets are balanced across its instances using round-robin balancing. More details are available in the Networking section of the manuals. 
+{{< /callout >}}
+
+{{< callout context="tip" title="Oakestra Dashboard" icon="outline/rocket">}}
+  You can choose the IPv4 addresses for each service arbitrarily. In your deployment descriptor, you can type an IP address in the form `10.30.X.Y` next to the associated balancing policy. In this example,` "rr_ip": "10.30.55.55"`  binds `10.30.55.55` to the round-robin policy for that service. Every time we perform a request towards that address, the packets are balanced across its instances using round-robin balancing. More details are available in the Networking section of the manuals. 
+{{< /callout >}}
```

Comment:
Same repetition issue. I believe the IDE adds copy of the text if you do inline resolution.